db/migrate/20151015142633_add_role_column.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleColumn < ActiveRecord::Migration
+  def change
+    add_column :users, :role, :string, null: false, default: 'Member'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.